gpt3discord.py

```diff
@@ -5,24 +5,21 @@
 
 import discord
 from discord.ext import commands
-from dotenv import load_dotenv
 import os
 
 if sys.platform == "win32":
     separator = "\\"
 else:
     separator = "/"
 
-print("The environment file is located at " + os.getcwd() + separator + ".env")
-load_dotenv(dotenv_path=os.getcwd() + separator + ".env")

```

Comment:
Taking this out of global scope is a good thing. Was on my list of suggestions. Thanks.